cmd/kube-controller-manager/app/plugins_providers.go

```diff
@@ -28,11 +32,20 @@ import (
 	"k8s.io/kubernetes/pkg/volume/vsphere_volume"
 )
 
+type probeFn func() []volume.VolumePlugin
+
+func probePluginWithCSIMigrationInTreeOff(plugins []volume.VolumePlugin, key featuregate.Feature, fn probeFn) []volume.VolumePlugin {
```

Comment:
this could probably just be renamed `ProbePluginsConditional` or something like that. Since the current name implies that `CSIMigrationInTreeOff==true`